aio: implement remark renderer

petebacondarwin opened the pull request.

mary-poppins commented:
The angular.io preview for c13848d is available [here][1].

[1]: https://pr15863-c13848dec06fe5ced26cd53a7d78c9179748fd8d.ngbuilds.io/

mary-poppins commented:
The angular.io preview for b9c3673 is available [here][1].

[1]: https://pr15863-b9c367302b8740b3e250d4fada114a965c31aff2.ngbuilds.io/

Foxandxss commented on aio/content/examples/cli-quickstart/src/styles.css:
```diff
@@ -0,0 +1 @@
+undefined
```
this should not be undefined. Be sure you have this file.

petebacondarwin replied:
@Foxandxss - so I pulled from angular.io master. Are you saying that I need to generate the boilerplate before running migration?

Foxandxss replied:
No, that file should exist already.

petebacondarwin replied:
hmm then this is a bug in the migrator...

petebacondarwin committed:
docs(aio): update to remark syntax

mary-poppins commented:
The angular.io preview for e6e5b6e is available [here][1].

[1]: https://pr15863-e6e5b6ec2b6bf232d95cbe094736528e9946d574.ngbuilds.io/

petebacondarwin committed:
build(aio): remove unused Rho package

mary-poppins commented:
The angular.io preview for 99f7c6b is available [here][1].

[1]: https://pr15863-99f7c6b2dfdbde5485e0496d17b97ddbf1cad9e3.ngbuilds.io/

petebacondarwin committed:
build(aio): implement the remark renderer

The implementation adds three plugins to the remark processor:

* remove support for code blocks triggered by indented
text - only gfm triple backticks are supported; and also adds support for
dgeni inline tags.

* ignore content within `code-example` and `code-tabs` elements. This prevents
the content being accidentally treated as markdown

* ignore dgeni inline tags, e.g. `{@link ... }` to prevent the content of
the links from being accidentally treated as markdown

mary-poppins commented:
The angular.io preview for a41a90b is available [here][1].

[1]: https://pr15863-a41a90b15b254879240eacff3f6821431d5613d7.ngbuilds.io/

petebacondarwin committed:
docs(aio): revert file-trees back to original structure

This is an auto-migration from angular.io

mary-poppins commented:
The angular.io preview for 0f239c9 is available [here][1].

[1]: https://pr15863-0f239c979311edd51b577e3fff9ad7f17e6620c4.ngbuilds.io/

mary-poppins commented:
The angular.io preview for 161e556 is available [here][1].

[1]: https://pr15863-161e55675b728cda49e49876d963dd526c3a56f0.ngbuilds.io/

mary-poppins commented:
The angular.io preview for 71e0aca is available [here][1].

[1]: https://pr15863-71e0aca355a779d70c00ec2a4142c509fd72f70d.ngbuilds.io/

mary-poppins commented:
The angular.io preview for e1cf1a0 is available [here][1].

[1]: https://pr15863-e1cf1a0a9c2e93a0945b3ca650be3cee8853661a.ngbuilds.io/
